experiments/ClimaEarth/run_amip.jl

```diff
@@ -756,6 +718,11 @@ function solve_coupler!(cs)
 
         ## callback to checkpoint model state
         TimeManager.trigger_callback!(cs, cs.callbacks.checkpoint)
+
+        ## compute/output AMIP diagnostics if scheduled for this timestep
+        ## wrap the current CoupledSimulation fields and time in a NamedTuple to match the ClimaDiagnostics interface
+        cs_nt = (; out = nothing, u = cs.fields, p = nothing, t = cs.t, step = Δt_cpl)
```

Comment:
`step` here is the number of iteration we are at, ie an integer counter. In this case, it is roughly cs.t / dt_cpl

Reply:
`out = nothing` is also not needed